src/gz21_ocean_momentum/testing/main.py

```diff
@@ -152,9 +153,11 @@
 # Read the dataset file
 print("loading dataset...")
 xr_dataset = xr.open_zarr(data_file)
-if input("global?").lower() == "y":
-    # This will add a cyclic transform when used on our model
-    xr_dataset.attrs["cycle"] = 360
+
+# TODO bring back the following lines
+#if input("global? (y / n)").lower() == "y":
+#    # This will add a cyclic transform when used on our model
+#    xr_dataset.attrs["cycle"] = 360
 
 with ProgressBar(), TaskInfo("Applying transforms to dataset"):
     xr_dataset = submodel.fit_transform(xr_dataset)
```

Comment:
Line 170: Why is there the option to do training again? In which case would we use that?

Reply:
OK, keeping it in in case needed later